.github/workflows/build.yml

```diff
@@ -59,6 +60,26 @@ jobs:
       # This step uses the `docker/build-push-action` action to build the image, based on your repository's `Dockerfile`. If the build succeeds, it pushes the image to GitHub Packages.
       # It uses the `context` parameter to define the build's context as the set of files located in the specified path. For more information, see "[Usage](https://github.com/docker/build-push-action#usage)" in the README of the `docker/build-push-action` repository.
       # It uses the `tags` and `labels` parameters to tag and label the image with the output from the "meta" step.
+      - name: Build and export to Docker
+        uses: docker/build-push-action@v6
+        with:
+          load: true
+          context: api-implementation
+          build-args: |
+            PYTHON_VERSION=${{ steps.python-version.outputs.PYTHON_VERSION }}
+            SERVICE_PATH=src/mpic_${{ matrix.service }}_service
+          file: api-implementation/Dockerfile
+          tags: ${{ env.REGISTRY }}/${{ github.repository_owner }}/${{ matrix.service }}:${{ env.TEST_TAG }}
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@0.28.0
+        with:
+          image-ref: ${{ env.REGISTRY }}/${{ github.repository_owner }}/${{ matrix.service }}:${{ env.TEST_TAG }}
+          format: 'table'
```

Comment:
We might be able to have the results appear in the GitHub UI if we use this format
```suggestion
          format: 'github'
```